config/src/config/json_rpc_config.rs

```diff
@@ -22,7 +22,7 @@ pub const DEFAULT_JSON_RPC_ADDRESS: &str = "127.0.0.1";
 pub const DEFAULT_JSON_RPC_PORT: u16 = 8080;
 pub const DEFAULT_BATCH_SIZE_LIMIT: u16 = 20;
 pub const DEFAULT_PAGE_SIZE_LIMIT: u16 = 1000;
-pub const DEFAULT_CONTENT_LENGTH_LIMIT: usize = 128 * 1024; // 128kb
+pub const DEFAULT_CONTENT_LENGTH_LIMIT: usize = 1024 * 1024; // 1mb
```

Comment:
Instead of changing this here from 4mb to 1mb you can change the size of request in the test thats failing for unknown reasons. Through testing `4194304` seems to be a size that gets a rejection but doesn't cause the OS error we were seeing

Reply:
I will update it if landing blocking test failed again :( otherwise, will submit a new PR